packages/server/src/internalHandlers/account.ts

```diff
@@ -61,6 +61,7 @@ export const request: RPCHandler<AccountRequest["result"]> = async (
   const account = await walletHandler({
     currencies$: filteredCurrencies$,
     accounts$: filteredAccounts$,
+    showAccountFilter: !!showAccountFilter,
```

Comment:
It should already be parsed as a boolean by zod above
```suggestion
    showAccountFilter,
```